diff -Naur splashutils-1.5.4.4/src/libfbsplash.c splashutils-1.5.4.4-m/src/libfbsplash.c
--- splashutils-1.5.4.4/src/libfbsplash.c	2011-06-13 01:43:39.000000000 +0400
+++ splashutils-1.5.4.4-m/src/libfbsplash.c	2017-03-25 11:56:57.080233695 +0300
@@ -495,7 +495,7 @@
 
 nosave:
 	/* Clear a stale mtab entry that might have been created by the initscripts. */
-	system("/bin/sed -i -e '\\#"FBSPLASH_CACHEDIR"# d' /etc/mtab");
+//	system("/bin/sed -i -e '\\#"FBSPLASH_CACHEDIR"# d' /etc/mtab");
 
 	umount2(what, MNT_DETACH);
 	return err;
